docs/devices/324131092621.md

```diff
@@ -74,6 +74,9 @@ This device supports OTA updates, for more information see [OTA updates](../guid
 
 * `legacy`: Set to false to disable the legacy integration (highly recommended), will change structure of the published payload (default true). The value must be `true` or `false`
 
+  - `legacy` mode will allow you to track the number of button presses - which are exposed under the `counter`field. The button presses must be `.25` apart to register.
```

Comment:
This should be moved below `<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->`, but I don't think we should mention any legacy features here (we don't want people to use them anymore)

Reply:
@Koenkk  - fair enough ... however the main issue would be that the existing Hue Apps handle this in software. Additionally ZHA apparently also has it handled in software. So when you come to Z2M one would be confused as to why a feature we "thought" was part of the remote is missing..

Ive seen some issues about it as well..
https://github.com/Koenkk/zigbee2mqtt/issues/10312


Happy to close the PR -> but - I do think perhaps somewhere in the docs this should be mentioned... (in a different way if you prefer...)

Reply:
Is there a better place to mention somewhere in the docs for the various systems that multi-click is not an actual feature of the remote but has to be implemented in software somewhere?

Thanks!

Reply:
Maybe the FAQ?

Reply:
You can create a feature request [here](https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=feature+request&template=feature_request.yaml&title=%5BFeature+request%5D%3A+), if more people are interested in it, it can be implemented.